package.json

```diff
@@ -23,36 +51,50 @@
     "ignore": "^5.2.4"
   },
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.18.2",
     "@babel/core": "^7.22.9",
     "@babel/eslint-parser": "^7.22.9",
-    "@eslint-community/eslint-plugin-mysticatea": "^15.5.1",
-    "@eslint/core": "^0.13.0",
+    "@eslint-community/eslint-plugin-mysticatea": "^15.8.0",
+    "@eslint/core": "^1.0.0",
     "@eslint/css": "^0.6.0",
-    "@types/node": "^14.18.54",
+    "@types/cross-spawn": "^6.0.6",
+    "@types/eslint": "^8",
+    "@types/mocha": "^10.0.10",
+    "@types/node": "^24.10.1",
+    "@types/rimraf": "^3.0.2",
+    "@types/semver": "^7.7.1",
     "@vuepress/plugin-pwa": "^1.9.9",
     "cross-spawn": "^7.0.3",
-    "esbuild": "^0.19.3",
-    "eslint": "^8.46.0",
+    "esbuild": "^0.27.0",
+    "eslint": "^8.57.1",
+    "eslint-v9": "npm:eslint@^9.39.1",
+    "expect-type": "^1.2.2",
     "fs-extra": "^10.1.0",
     "mocha": "^10.4.0",
     "monaco-editor": "^0.47.0",
     "nyc": "^15.1.0",
     "opener": "^1.5.2",
     "rimraf": "^3.0.2",
-    "semver": "^7.5.4",
-    "vite-plugin-eslint4b": "^0.2.1",
+    "semver": "^7.7.3",
+    "tsdown": "^0.16.6",
+    "typescript": "^5.9.3",
+    "vite-plugin-eslint4b": "^0.6.0",
     "vitepress": "^1.0.0-rc.15"
   },
   "scripts": {
+    "build": "rimraf dist/ && tsdown --config-loader='unrun' --config=tsdown.config.mts",
+    "prepack": "npm run clean && npm run build",
     "preversion": "npm test",
     "version": "node scripts/update && git add .",
     "postversion": "git push && git push --tags",
-    "clean": "rimraf .nyc_output coverage docs/.vitepress/cache",
+    "clean": "rimraf .nyc_output/ coverage/ docs/.vitepress/cache/ dist/",
```

Comment:
Why not just use `clean: true` in the tsdown config (the default)?  Then you don't have to call clean or rimraf before build explicitly.

Reply:
Makes sense. Thanks for pointing that out.  It seems like their recommendation is to use `clean` for one of the configs, rather than both.  Would that be cleaner than doing it on the script side?